Source/Core/Core/HW/DVDInterface.cpp

```diff
@@ -1291,7 +1343,7 @@ u64 SimulateDiscReadTime(u64 offset, u32 length)
 		}
 	}
 
-	g_last_read_offset = (offset + length - 2048) & ~2047;
+	g_last_read_offset = end_offset;
```

Comment:
Okay, so the current code essentially has the same problem as the `if (decrypt) ...` code. I think modifying the code for buffered reads would be better than keeping the "- 2048" here.

To clarify, does the hardware only let you use the buffer if you're reading within 2048 bytes back, or is it possible to read from the buffer further back than that too?